extensions/amp-consent/0.1/consent-info.js

```diff
@@ -117,6 +122,7 @@ export function getStoredConsentInfo(value) {
       CONSENT_ITEM_STATE.UNKNOWN,
       undefined,
       undefined,
+      undefined,
       undefined
```

Comment:
You don't need to do this. If an argument is not specified, it's implied as `undefined`.